app/vmalert/unittest/testdata/long-period.yaml

```diff
@@ -0,0 +1,28 @@
+evaluation_interval: 100d
+
+rule_files:
+  - rules.yaml
```

Comment:
can we load rules from URL or S3 here? https://docs.victoriametrics.com/vmalert.html#reading-rules-from-object-storage

Reply:
I think so, use the same [Parse](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/app/vmalert/config/config.go#L221) func here.

Reply:
Cool! So I'm thinking maybe we should load and parse files/configs before we actually run unit tests? The reasoning is the following:
1. Parsing all configured test files before actually running them would fail fast if there are problems
2. Pre-loading files from URL or S3 could reveal config issues faster without waiting for sequential processing

Reply:
We did parse the file/configs first, then verify `input_series` and write data if configs are all correct.
https://github.com/VictoriaMetrics/VictoriaMetrics/blob/c6dca2719183f895b09f237e62615aa83edf4873/app/vmalert/unittest.go#L216
https://github.com/VictoriaMetrics/VictoriaMetrics/blob/c6dca2719183f895b09f237e62615aa83edf4873/app/vmalert/unittest.go#L224

>Parsing all configured test files before actually running them would fail fast if there are problems

I didn't parse all files at the begining though, now if one test file config is wrong, error will be print and other tests will still run, I thought it's more friendly.